README.md

```diff
@@ -58,23 +56,23 @@ To build a Ruby image, choose either the CentOS or RHEL based image:
     ```
     $ git clone --recursive https://github.com/sclorg/s2i-ruby-container.git
     $ cd s2i-ruby-container
-    $ make build TARGET=rhel9 VERSIONS=3.0
+    $ make build TARGET=rhel9 VERSIONS=3.3
     ```
 
 *  **CentOS Stream based image**
 
     This image is available on DockerHub. To download it run:
 
     ```
-    $ podman pull quay.io/sclorg/ruby-30-c9s
+    $ podman pull quay.io/sclorg/ruby-33-c9s
     ```
 
     To build a Ruby image from scratch run:
 
     ```
     $ git clone --recursive https://github.com/sclorg/s2i-ruby-container.git
     $ cd s2i-ruby-container
-    $ make build TARGET=c9s VERSIONS=3.0
+    $ make build TARGET=c9s VERSIONS=3.3
     ```
```

Comment:
RHEL 9 and c9s instructions should retain 3.0 for less updates going into the future. 3.3 is shorter-lived stream compared to 3.0 which is supported for the RHEL 9 lifetime.

Reply:
Fixed by https://github.com/sclorg/s2i-ruby-container/pull/580/commits/66bf29c3473b78fdb6734dcb7bb2de57cd22ba87